environments/shared/recommended.js

```diff
@@ -663,6 +663,12 @@ module.exports = {
 
     // Forbid a module from importing itself
     'import/no-self-import': 'error',
+
+    // Enforce a convention in the order of require() / import statements
+    'import/order': ['error', {
```

Comment:
This is stylistic rule and therefore must not cause build failures (randomly-ordered imports do not imply broken code).

Please change to `warn` and move to `coding-styles/fixable`, thanks! 🙏